5.13.2

silasary opened the pull request:
July Update?

silasary committed:
Fix typo in pyproject.toml (#1711)

AstreaTSS committed:
fix: fallback to list if entitlements is not provided (#1715)

silasary committed:
feat: Add audioop-lts for python 3.13+ (#1713)

Fixes #1685

sondevra committed:
fix: command lookup error when scope is cached (#1716)

* fix: command lookup error when scope is cached

Fixed lookup to use .get() on the dictionary to prevent large log of error in the circumstance of a cache command showing on a changed scope

Signed-off-by: Sondra <125593620+sondevra@users.noreply.github.com>

* Update context.py

Signed-off-by: Sondra <125593620+sondevra@users.noreply.github.com>

---------

Signed-off-by: Sondra <125593620+sondevra@users.noreply.github.com>

AstreaTSS committed:
fix/test: ensure polls are being correctly processed (#1714)

* fix: use answer_id from data, not options

* fix: correctly deserialize question for polls

* ci: add tests for polls

* test: make poll dict test more resilient

mifuyutsuki committed:
fix: change CronTrigger next_fire start time (#1718)

Replaces the croniter start_time in CronTrigger from `datetime.now()` to `self.last_call_time.astimezone()` to fix a bug causing possible multiple firing.

Fixes #1717

AstreaTSS commented:
👋. I'm going to try to get this moving along.

Current fix PRs: #1724, #1725, #1727, #1728, #1729, #1730 (technically a feat PR, but the lack of the new feature causes a lot of annoying errors so why not)

I'll be reviewing what I can today, although some of these are my PRs and so will require another person to help out.

retr0-init committed:
fixed: message response_metadata user id (#1729)

retr0-init committed:
feat: Add poll result embed type (#1730)

Lacosst0 committed:
fix: make _on_raw_message_poll_vote events use correct data (#1725)

SirTurlock committed:
fix: use int nonce for heartbeat instead of float for VoiceGateway (#1727)

Fixes #1726
VoiceGateway no longer accepts a string or a float in the heartbeat nonce and instead immediately disconnects the client with code 4020.
Fixed send_heartbeat to use an uint64 instead of a float for the nonce.

AstreaTSS commented:
All not-me PRs have been merged. Only #1724 and #1728 remain.

AstreaTSS committed:
fix: use localizations for InteractionCommand (#1724)

AstreaTSS committed:
fix: correctly serialize slash cmd with same names from different scope (#1733)

AstreaTSS committed:
fix: properly parse aliases for base commands (#1728)

* fix: properly parse aliases for base commands

* fix: don't add random aliases to base commands

LawMixer reviewed (approved):
Forgot to review these changes - they work for me on Python3.10 and Python3.11;

silasary committed:
chore: Bump version to 5.13.2
